apps/nextjs-app/src/features/app/blocks/view/grid/GridViewBase.tsx

```diff
@@ -287,11 +289,20 @@ export const GridViewBase: React.FC<IGridViewProps> = (props: IGridViewProps) =>
           neighborRecords[1] = rowStart >= realRowCount - 1 ? null : recordMap[rowStart + 1];
         }
 
+        const selectedRecordCount = isCellSelection
+          ? rowEnd - rowStart + 1
```

Comment:
`rowStart` and `rowEnd` denote the start and end rows of a selection, with `rowStart` possibly exceeding `rowEnd`